src/routes/spaces/address-books.controller.spec.ts

```diff
@@ -238,4 +538,35 @@ describe('AddressBooksController', () => {
       .expect(201);
     return { memberAccessToken };
   };
+
+  const createAddressBookItem = async (args: {
```

Comment:
Nit: based on the areas that this can be reused, let's rename this to `upsertAddressBookItem`.

```suggestion
  const upsertAddressBookItem = async (args: {
```